lib/datapacksutils.js

```diff
@@ -1305,7 +1309,11 @@ DataPacksUtils.prototype.getFieldDiffsIndividual = async function(allSourceDataP
                 var allTargetSObjects = [];
 
                 if (targetDataPack) {
-                    allTargetSObjects = this.getAllIndividualSObjectsInDataPack(targetDataPack.VlocityDataPackData[dataField][0]);
+                    if(targetDataPack.VlocityDataPackData[dataField]){
```

Comment:
Would be better to just call this.getDataField(targetDataPack); again for targetDataPack